src/js/components/Fragments/Navbuttons.scss

```diff
@@ -0,0 +1,35 @@
+.Navbuttons{
+    position: absolute;
+    bottom: 20px;
+    width: 100%;
+    overflow: hidden;
+    max-height: 60px;
+    background-color: #eee0;
```

Comment:
Does this background give you the desired color?
#eee seems to give a cream white color but with zero at the end, it reverts back to the default color.

Reply:
It is to set the background to be transparent, because opacity would set even the content within the div to transparent, but this method set only the background to blank and it's children to be visible

Reply:
use `background-color: transparent` instead

Reply:
Thank you I have used that